web/frontend/src/main-lib.ts

```diff
@@ -12,5 +12,8 @@ export const createRcApp = (
     webrtcEnabled: boolean;
     signalingAddress: string;
     overrides?: RCOverrides;
+    hiddenSubtypes?: string[];
+    hideDoCommand?: boolean;
+    hideOperationsSessions?: boolean;
```

Comment:
This is ugly but going away soon, so I will stick with this pattern. `ComponentProps<RemoteControlCards>` complains about unsafe any assignment when using the component constructor. Not worth messing with it.